addon/templates/components/materialize-collection-item.hbs

```diff
@@ -0,0 +1,13 @@
+{{#if isHeader}}
+  <h4>{{yield}}</h4>
+{{else}}
+  {{#if image}}
+    <img class="circle" alt="{{imageAlt}}" src="{{image}}"/>
+  {{/if}}
+  {{#if title}}
+    <span class="title">{{title}}</span>
+    <p>{{yield}}</p>
+  {{else}}
+    {{yield}}
+  {{/if}}
+{{/if}}
```

Comment:
I wonder if there's a more efficient way to go about this? When thinking about the 1000-items-in-a-collection use case, it may be prudent to make this as lightweight as possible.